aip_x2_gen2_launch/launch/topic_state_monitor.launch.py

```diff
@@ -60,7 +60,7 @@ def generate_launch_description():
 
     # Radar topic monitors
     radar_front_center_monitor = ComposableNode(
-        package="topic_state_monitor",
+        package="autoware_topic_state_monitor",
         plugin="topic_state_monitor::TopicStateMonitorNode",
```

Comment:
This plugin needs `autoware::` namespace. Thus please fix from `topic_state_monitor::TopicStateMonitorNode` to `autoware::topic_state_monitor::TopicStateMonitorNode` and other places too.

Reply:
fixed in https://github.com/tier4/aip_launcher/pull/375/commits/d74f504294d1fb6f3170584ba624bb363fbfaed4